includes/widgets/video.php

```diff
@@ -822,7 +830,11 @@ protected function render() {
 		<div <?php echo $this->get_render_attribute_string( 'video-wrapper' ); ?>>
 			<?php
 			if ( ! $settings['lightbox'] ) {
-				echo $video_html; // XSS ok.
+				if ( 'youtube' === $settings['video_type'] ) {
```

Comment:
You shouldn't fetch the video html when it's a YouTube video

Reply:
don't understand

Reply:
Do the condition above